packages/side-nav/src/vaadin-side-nav-item.js

```diff
@@ -94,6 +95,14 @@ class SideNavItem extends ElementMixin(ThemableMixin(PolylitMixin(LitElement)))
        */
       path: String,
 
+      /**
+       * A comma-separated list of alternative paths matching this item.
```

Comment:
Let's add JSDoc annotation to this comment as well.

Let's add JSDoc annotation for Lit analyzer to use dash-case attribute:

```
@attr {string} path-aliases
```

Reply:
Done.